pkgs/servers/mail/dovecot/default.nix

```diff
@@ -142,12 +143,11 @@ stdenv.mkDerivation rec {
       # We need this so utilities default to reading /etc/dovecot/dovecot.conf file.
       "--sysconfdir=/etc"
       "--with-moduledir=${placeholder "out"}/lib/dovecot/modules"
-      "--with-ldap"
+      "--with-ldap=${if withLDAP then "yes" else "no"}"
```

Comment:
I would use `lib.optionals` instead of using `yes` or `no`.

Reply:
Fixed!